City-of-Helsinki/kukkuu-admin
# KK-613, KK-646 | Add create and edit views for event groups

## Description

Adds both create and edit views for event groups. The edit view also contains the delete button.

Both of the views rely on the `EventGroupForm`. By having all of these features implemented, I was able to create a browser test that covers the rest of the CRUD operations.

## Context




[KK-613](https://helsinkisolutionoffice.atlassian.net/browse/KK-613)
[KK-646](https://helsinkisolutionoffice.atlassian.net/browse/KK-646)

## How Has This Been Tested?

With a browser test that checks that creation, editing and deletion works.

## Manual Testing Instructions for Reviewers

As a logged in user

1. Go to event list
1. Click even group creation button
1. Expect name to be required
1. Expect the Finnish name to be required
1. Expect saving to work when requirements are met
1. Select the event group you created
1. Click edit button
1. Save field/fields
1. Expect saving to work
1. Select the event group you edited
1. Click edit button
1. Click delete button
1. Click confirm
1. Expect event group to be deleted
